missilewars-plugin/src/main/java/de/butzlabben/missilewars/util/PlayerEquipmentRandomizer.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * This file is part of MissileWars (https://github.com/Butzlabben/missilewars).
+ * Copyright (c) 2018-2021 Daniel Nägele.
+ *
+ * MissileWars is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MissileWars is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MissileWars.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.butzlabben.missilewars.util;
+
+import de.butzlabben.missilewars.Logger;
+import de.butzlabben.missilewars.game.Game;
+import de.butzlabben.missilewars.wrapper.abstracts.Arena;
+import de.butzlabben.missilewars.wrapper.missile.Missile;
+import de.butzlabben.missilewars.wrapper.player.MWPlayer;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.Collections;
+
+/**
+ * @author Butzlabben
+ * @since 19.01.2018
+ */
+
+public class PlayerEquipmentRandomizer {
+
+    private final MWPlayer mwPlayer;
+    private final Game game;
+    private final Arena arena;
+
+    private final int maxGameDuration;
+    private final int defaultIntervalByTeamAmount = 20;
+    private final int defaultFactorByGameTime = 1;
```

Comment:
please make these constants, so also static and write thus in UPPER_SNAKE_CASE
```suggestion
    private static final int DEFAULT_INTERVAL_BY_TEAM_AMOUNT = 20;
    private static final int DEFAULT_FACTOR_BY_GAME_TIME = 1;
```